### ISSUE TITLE
File Selector disappears

### ISSUE THREAD

**Issue by robd003** [transmission/transmission#3441]
Using build [`5fc9f22fab` ](https://github.com/transmission/transmission/commit/5fc9f22fabd87c5676c9653e1069d9c388c431cc) has an issue where if you open the Torrent Inspector, select a file and then move to a different torrent the file section will be blank from then on.

I've tried closing / re-opening the Torrent Inspector but it just totally loses the Files selection component and shows up blank.

This may only happen if you use the Filter input box and then move to a different torrent.

<img width="713" alt="Screen Shot 2022-07-12 at 11 26 16 PM" src="https://user-images.githubusercontent.com/571328/178644235-21702b45-3e6c-4b7c-8aa1-e64554709e8f.png">

**Comment by robd003** [comment]
@GaryElshaw @Coeur anything I can do to help provide debugging info for this?

**Comment by GaryElshaw** [comment]
Sorry Rob, I can't help with this one. It hopefully might be fixed when @sweetppro's pr lands.

**Comment by Coeur** [comment]
I have no time to look at this, I'm afraid.

**Comment by MehdiSaffar** [comment]
I am using Transmission 4.0.0-beta.1 (afdae24995) on macOS Monterey 12.6, I have a similar issue where the files inspector is empty. I did not do any file moving or anything like described in the original post. All I did was open Transmission, start a download using a magnet link, then double clicked to open the inspector.

<img width="595" alt="image" src="https://user-images.githubusercontent.com/13124263/194566772-f2df9412-99a6-4077-9a7d-f8fe97dd384e.png">
